Prevent test-running a program as root during the install (which fails).
--- Makefile.am.orig	2017-08-22 09:56:33.000000000 -0500
+++ Makefile.am	2022-03-22 20:32:08.000000000 -0500
@@ -50,7 +50,6 @@
 		 || chmod 664 "$(DESTDIR)$(scoredir)/mbscore" ; } \
 	    && chmod 575 "$(DESTDIR)$(scoredir)" ; \
 	fi
-	$(DESTDIR)$(bindir)/moon-buggy -c
 
 uninstall-local:
 	rm -f $(DESTDIR)$(scoredir)/mbscore
